json-endpoints/class.wpcom-json-api-get-comments-tree-endpoint.php

```diff
@@ -32,7 +32,7 @@ function intval_array_map( $comments ) {
 			return array_map( 'intval', $comments );
 		}
 
-		return array( $total_count, intval_array_map( $db_comments ) );
+		return array( $total_count, array_map( 'intval_array_map', $db_comments ) );
```

Comment:
maybe we could give a more meaningful name to what our results should be…something like `get_comment_ids`